### PR TITLE
fix(setup): add Python 2.7 compatibility + fix file not found error

### PR DESCRIPTION
**Pull request by JulienGrv**
I made the script PEP8 compliant at the same time

### CODE REVIEW

**Comment by NiklasRosenstein** [comment]
Thanks!

Could you please revert the 4-space indentation back to 2-space and the print in line 11-12 back on one line?

**Review by NiklasRosenstein — CHANGES_REQUESTED** [review]
As already noted above.

**Comment by JulienGrv** [comment]
2 spaces is actually not PEP8 compliant but I reverted back to 2 spaces anyway

File: setup.py [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
 
+
 def restify():
   if os.path.isfile('README.md'):
     if os.system('pandoc -s README.md -o README.rst') != 0:
-      print('-----------------------------------------------------------')
-      print('WARNING: pandoc command failed, could not restify README.md')
-      print('-----------------------------------------------------------')
+      print('----------------------------------------------------------')
+      print(
```
Comment (NiklasRosenstein) [inline setup.py]: Please also put this on one line.

**Comment by NiklasRosenstein** [comment]
Looks good, thanks!

**Comment by NiklasRosenstein** [comment]
Woops! I closed it instead of merging first.

**Pull request merged**